### ISSUE TITLE
Live preview while generating is gone

### ISSUE THREAD

**Issue by tripletto** [vladmandic/automatic#213]
### Issue Description

Live preview while generating is gone. Must have happend somewhere after **commit e8d8dae: fix broken js** because I am pretty sure it worked in that commit.

**Comment by vladmandic** [comment]
i don't have any issues with it. double check if they are enabled? ui -> settings -> live preview -> 
set `Show new live preview image every N sampling steps` to 1

**Comment by tripletto** [comment]
That's odd. Tried Edge and Firefox and behold: Live preview works as intended.

Chrome not so much even disabling all extensions doesn't help. BUT Chrome in private mode works.

But you can probably close this then. Seems to be something on my or Chrome's end. I'll use FF for now.

**Comment by vladmandic** [comment]
clear browser cache? to confirm, open browser inspector and in network tab select 'disable cache'

**Comment by mart-hill** [comment]
I can confirm, in the `Version: 15d62844 Tue Apr 18 12:11:02 2023 -0400`, on newest Edge,  I don't get the live preview anymore, with these settings:
<img width="1668" alt="image" src="https://user-images.githubusercontent.com/55754730/232863008-872a68b2-9ef0-46e5-adcd-96b04c3c2d91.png">

**Comment by vladmandic** [comment]
anytime i see 

```
> Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'props')
    at Array.forEach ()
```

it typically means that input vs output gradio components don't match expected ones.
can you go to settings -> apply
and then restart server - that typically populates missing items

**Comment by cjps-br** [comment]
I don't see the previews nor the progressbar in Brave v1.50.119 either.
And when I press the generate button, the buttons to interrupt the generation don't appear.

**Comment by mart-hill** [comment]
That, too, indeed.

**Comment by cjps-br** [comment]
I was using the monochrome skin and the buttons didn't show up. I put the black/orange skin and the buttons appear. But it still doesn't generate the live preview
![Uploading Captura de tela 2023-04-18 172859.jpg…]()

**Comment by EfourC** [comment]
I just tried this repo for the first time yesterday (so many performance issues fixed for me!!) and live preview was working fine.

Just tried today, and find it's not working.

Browser:  ungoogled-chromium

**Comment by Anipsy** [comment]
Can confirm that with [2b1b75dd](https://github.com/vladmandic/automatic/commit/2b1b75dd12b9bc33eb8ad1f02bb9f026b14032b5) commit as of today preview if gone for me as well, i've tried in firefox and chrome with cache reset and changed preview on and off in UI settings.  

Also "Generate" button doesn't go gray anymore while its generating, i'm using default gradio theme to make sure it's not a theme issue.

**Comment by Panchvzluck** [comment]
Can confirm live previews don't work now, using Chorme.

**Comment by vladmandic** [comment]
Just to confirm, this affects any sampler?

**Comment by SqueakyZee** [comment]
In my experience, yes, all the samplers are being affected

**Comment by FutonGama** [comment]
Its not working here too

**Comment by iDeNoh** [comment]
Every sampler I've tried

**Comment by iDeNoh** [comment]
> Just to confirm, this affects any sampler?

All of the samplers I've tried so far, yes.

**Comment by FutonGama** [comment]
I figured out. The problem is on progressbar.js inside Javascript folder. You guys can use the old one for now until it as fixed. Here the link: https://www.mediafire.com/file/g91a7ncnspshxpg/old+progressbar+for+now.rar/file

**Comment by arcum42** [comment]
Yeah, if the default value for the parameter "once" in requestProgress (in progressbar.js) is false, I'm pretty sure it shouldn't actually be passed the text prompt you entered as a value instead...

**Comment by MADxingjin** [comment]
Confirm Issue still exists on 
Firefox 112.0.1,
MS Edge 112.0.1722.48 ,
Google Chrome 112.0.5615.122 
At commit  `2b1b75dd Tue Apr 18 14:56:19 2023 -0400`

**Comment by vladmandic** [comment]
ok, i'm back online - this last hint helped. issue was that gradio is sending extra params to js function, so order of params got messed up - and i introduced the issue when i added feature to reconnect browser on restart.
i've just implemented reconnect slightly differently so params sent by gradio don't matter.
fixed.
